### PR TITLE
Lesson 5 pull request

### PR DESCRIPTION
**Pull request by PYeremeyev**
Added lesson 5 screenshot for theory
Added program (triangle exists?) + Tests

### CODE REVIEW

File: ASCH_Practice/src/test/java/TriangleExistsTest.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import org.junit.*;
+import org.junit.jupiter.api.BeforeAll;
```
Comment (AShnyakin) [inline ASCH_Practice/src/test/java/TriangleExistsTest.java]: redundant import

File: ASCH_Practice/src/test/java/TriangleExistsTest.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import org.junit.*;
+import org.junit.jupiter.api.BeforeAll;
+
+import static org.junit.Assert.*;
+
+public class TriangleExistsTest {
+
+    TriangleExists t = new TriangleExists();
```
Comment (AShnyakin) [inline ASCH_Practice/src/test/java/TriangleExistsTest.java]: t is bad variable name, use sumething like triangleExists

File: ASCH_Practice/src/main/java/TriangleExists.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+public class TriangleExists {
```
Comment (AShnyakin) [inline ASCH_Practice/src/main/java/TriangleExists.java]: bad name for class, do not use verbs

File: ASCH_Practice/src/test/java/TriangleExistsTest.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+import org.junit.jupiter.api.BeforeAll;
+
+import static org.junit.Assert.*;
+
+public class TriangleExistsTest {
+
+    TriangleExists t = new TriangleExists();
+
+
+    @Test
```
Comment (AShnyakin) [inline ASCH_Practice/src/test/java/TriangleExistsTest.java]: where is checks for zero numbers

**Comment by PYeremeyev** [comment]
Fixed review comments

**Pull request merged**